pkg/controller/user-cluster-controller-manager/application-installer-controller/doc.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+Package applicationinstallercontroller contains a controller that is responsible for reconcile ApplicationInstallation
+(ie install, update or uninstall applications)
+*/
+package applicationinstallercontroller
```

Comment:
one general thing I just noticed. I think in the original proposal we called the controller "applicationinstallationcontroller". I think in order to avoid confusion, we should keep using that name